src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/Error.txt

```diff
@@ -163,103 +163,103 @@ Error({Kind:ErrorKind.EditPermissions})
 // ************* Default error messages *************
 
 >> IfError(Error({Kind:ErrorKind.None}), $"{FirstError.Kind}: {FirstError.Message}")
-"0: System no error"
+"0: System no error."
```

Comment:
>r. [](http://example.com/codeflow?start=18&length=2)

Those changes will need to be made in Power Apps, otherwise those tests will fail there. Or we leave the default error messages for those error kinds without a `.` - your call.

Reply:
Easy call. Ill roll them back.